### PR TITLE
chore: add continue_with documentation

### PR DESCRIPTION
**Pull request by jonas-jonas**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/self-service/flows/account-recovery-password-reset.mdx [hunk 1]
```diff
@@ -387,6 +387,60 @@ In this schema, you define both `email` and `email_secondary` fields as recovery
 messages to both addresses when the recovery flow is initiated. The user should provide the `email_secondary` field during
 registration or update it later in their account settings so it can be used in the recovery process.
 
+## Native recovery flows
+
+Ory Identities supports recovery flows in native applications. This allows you to build a native application that allows the user
+reset their password in the app without opening a browser. However, this flow is only available for the `code` method.
+
+Since enabling this alters the behavior of the native recovery flow, you need to enable it in the Ory Console or using the Ory
+CLI. The setting only affects the recovery flow for api clients. The other self-service flows and browser recovery flows are not
```
Comment (vinckr) [inline docs/kratos/self-service/flows/account-recovery-password-reset.mdx:396]:
```suggestion
CLI. The setting only affects the recovery flow for API clients. The other self-service flows and browser recovery flows are not
```

File: docs/kratos/self-service/flows/account-recovery-password-reset.mdx [hunk 2]
```diff
@@ -387,6 +387,60 @@ In this schema, you define both `email` and `email_secondary` fields as recovery
...
+affected.
+
+For new projects this setting is enabled by default.
```
Comment (vinckr) [inline docs/kratos/self-service/flows/account-recovery-password-reset.mdx:399]:
```suggestion
This setting is enabled by default for new projects.
```

File: docs/kratos/self-service/flows/account-recovery-password-reset.mdx [hunk 3]
```diff
@@ -387,6 +387,60 @@ In this schema, you define both `email` and `email_secondary` fields as recovery
 messages to both addresses when the recovery flow is initiated. The user should provide the `email_secondary` field during
 registration or update it later in their account settings so it can be used in the recovery process.
 
+## Native recovery flows
+
+Ory Identities supports recovery flows in native applications. This allows you to build a native application that allows the user
+reset their password in the app without opening a browser. However, this flow is only available for the `code` method.
```
Comment (vinckr) [inline docs/kratos/self-service/flows/account-recovery-password-reset.mdx:393]:
```suggestion
reset their password in the app without opening a browser. This flow is only available for the `code` method.
```

File: docs/kratos/self-service/flows/account-recovery-password-reset.mdx [hunk 4]
```diff
@@ -387,6 +387,60 @@ In this schema, you define both `email` and `email_secondary` fields as recovery
...
+[Ory Console](https://console.ory.sh/projects/current/recovery).
+
+![Native Recovery Flows](./assets/enable_native_recovery_flows.png)
+
+```mdx-code-block
+</TabItem>
+<TabItem value="cli" label="Ory CLI">
+```
+
+1. Download the Ory Identities config from your project and save it to a file:
```
Comment (vinckr) [inline docs/kratos/self-service/flows/account-recovery-password-reset.mdx:416]:
```suggestion
1. Download the Ory Identities configuration from your project and save it to a file:
```

please either config or configuration consistently.

**Review by vinckr — COMMENTED** [review]
lgtm, a few minor things

File: docs/kratos/self-service/flows/account-recovery-password-reset.mdx (hunk 4 above)
Comment (jonas-jonas) [reply docs/kratos/self-service/flows/account-recovery-password-reset.mdx:416]: Copy & pasted from above.

**Review by vinckr — APPROVED** [review]

**Pull request merged**